docs/configuration/resource-recommender.rst

```diff
@@ -0,0 +1,126 @@
+Cost Savings (KRR)
+************************************************************
+
+Robustas `KRR <https://github.com/robusta-dev/krr>`_ is a CLI tool for optimizing resource allocation in Kubernetes clusters.
+It gathers pod usage data from Prometheus and recommends requests and limits for CPU and memory.
+This reduces costs and improves performance.
+By default, every instance of Robusta that's connected to the UI will run a KRR scan on startup. Further KRR scans can be triggered in the UI, and all scans can be viewed there.
```

Comment:
Can you apply the same changes to this intro section 👇 as discussed in the popeye section above. (First overview then details.)